doc/sphinx/examples_python/comparison_to_saffman_fig13.py

```diff
@@ -17,54 +19,59 @@
     os.makedirs("./cache")
 cache = pi.MatrixElementCache("./cache")
 
+
 def setup_system_one(bfield):
     system_one = pi.SystemOne(state_one.getSpecies(), cache)
-    system_one.restrictEnergy(state_one.getEnergy()-100, state_one.getEnergy()+100)
-    system_one.restrictN(state_one.getN()-2, state_one.getN()+2)
-    system_one.restrictL(state_one.getL()-2, state_one.getL()+2)
+    system_one.restrictEnergy(state_one.getEnergy() - 100, state_one.getEnergy() + 100)
+    system_one.restrictN(state_one.getN() - 2, state_one.getN() + 2)
+    system_one.restrictL(state_one.getL() - 2, state_one.getL() + 2)
     system_one.setBfield([0, 0, bfield])
     return system_one
 
+
 def setup_system_two(system_one, angle):
     system_two = pi.SystemTwo(system_one, system_one, cache)
-    system_two.restrictEnergy(state_two.getEnergy()-5, state_two.getEnergy()+5)
+    system_two.restrictEnergy(state_two.getEnergy() - 5, state_two.getEnergy() + 5)
     system_two.setDistance(10)
     system_two.setAngle(angle)
-    if angle == 0: system_two.setConservedMomentaUnderRotation([int(2*state_one.getM())])
+    if angle == 0:
+        system_two.setConservedMomentaUnderRotation([int(2 * state_one.getM())])
     system_two.setConservedParityUnderInversion(pi.ODD)
     system_two.setConservedParityUnderPermutation(pi.ODD)
     return system_two
 
+
 def getEnergies(bfield, angle):
     # Set up one atom system
     system_one = setup_system_one(bfield)
     system_one.diagonalize(1e-3)
 
     # Calculate Zeeman shift
-    zeemanshift = 2*system_one.getHamiltonian().diagonal()[system_one.getBasisvectorIndex(state_one)] # GHz
+    zeemanshift = 2 * system_one.getHamiltonian().diagonal()[system_one.getBasisvectorIndex(state_one)]  # GHz
 
     # Set up two atom system
     system_two = setup_system_two(system_one, angle)
     system_two.diagonalize(1e-3)
 
     # Calculate blockade interaction
-    eigenenergies = (system_two.getHamiltonian().diagonal()-zeemanshift)*1e3 # MHz
+    eigenenergies = (system_two.getHamiltonian().diagonal() - zeemanshift) * 1e3  # MHz
     overlaps = system_two.getOverlap(state_two)
-    blockade = 1/np.sqrt(np.sum(overlaps/eigenenergies**2))
+    blockade = 1 / np.sqrt(np.sum(overlaps / eigenenergies**2))
 
     return blockade
 
-if __name__ ==  '__main__':
+
+if __name__ == "__main__":
     plt.xlabel(r"$B$ (Gauss)")
     plt.ylabel(r"Blockade (MHz)")
-    plt.xlim(-0.4,20.4)
-    plt.ylim(0,0.4)
+    plt.xlim(-0.4, 20.4)
+    plt.ylim(0, 0.4)
 
     with Pool() as pool:
         energies1 = pool.map(partial(getEnergies, angle=0), bfields)
-        energies2 = pool.map(partial(getEnergies, angle=np.pi/2), bfields)
+        energies2 = pool.map(partial(getEnergies, angle=np.pi / 2), bfields)
 
-    plt.plot(bfields, energies1, 'b-', label=r"$\theta = 0$")
-    plt.plot(bfields, energies2, 'g-', label=r"$\theta = \pi/2$")
-    plt.legend(loc=2);
```

Comment:
This trailing semicolon is intentional so that Jupyter doesn't write `<bla bla bla legend text>` into the output. Is it possible to add it back in without littering the tutorial notebooks with `# noqa` or so? Please also check the other notebooks for similar instances.

Reply:
This is a Python script. In the Jupyter notebooks, the trailing semicolons are still there.